tests/assert_utils.py

```diff
@@ -0,0 +1,25 @@
+import typing
+from os import PathLike
+from pathlib import Path
+from pprint import pformat
+
+
+def assert_file_exists(filepath: typing.Union[str, PathLike]) -> None:
+    """Assert if filepath points to an existing file. Print all the elements (files and dir) of the parent dir of the given filepath.
+
+    This is mostly to have better assert message than using a raw assert os.path.isfile(filepath).
+
+    Parameters
+    ----------
+    filepath : typing.Union[str, PathLike]
```

Comment:
you don't need to specify types in docstring if you did in typing.
```suggestion
    filepath
```

Reply:
I don't know, my IDE does it by default ; are you sure ?

Reply:
Yup.

Reply:
If you say so